packages/libs/react-components/src/utils/convertStyles.ts

```diff
@@ -0,0 +1,27 @@
+import { Primitive } from '../typings';
+
+export const convertStyles = (
+  styles: Record<string, Primitive | Record<string, Primitive>>,
+): Record<string, Primitive | Record<string, Primitive>> => {
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const newStyles = {} as any;
```

Comment:
can we use unknown here?

Reply:
it will break the function.
Albert and I took the best part of the morning to get this function properly typed. we failed. :/

Reply:
not needed anymore